tests/AddressTests.cpp

```diff
@@ -31,4 +31,9 @@ TEST(Address, ValidateLitecoin) {
     EXPECT_FALSE(validate("1BvBMSEYstWetqTFn5Au4m4GFg7xJaNVN2", TWCoinTypeLitecoin));
 }
 
+TEST(Address, ValidateBitcoinCash) {
+    EXPECT_TRUE(validate("bitcoincash:qruxj7zq6yzpdx8dld0e9hfvt7u47zrw9gfr5hy0vh", TWCoinTypeBitcoinCash));
+    EXPECT_FALSE(validate("1BvBMSEYstWetqTFn5Au4m4GFg7xJaNVN2", TWCoinTypeBitcoinCash));
```

Comment:
@hewigovens what was the case for bitcoin cash where user wanted to send to `1` address?

Reply:
This line should be true